worker/task/process_event_creation.go

```diff
@@ -221,6 +221,7 @@ func writeEventDeliveriesToQueue(ctx context.Context, subscriptions []datastore.
 			payload := EventDelivery{
 				EventDeliveryID: eventDelivery.UID,
 				ProjectID:       eventDelivery.ProjectID,
+				AcknowledgedAt:  eventDelivery.AcknowledgedAt,
```

Comment:
Why are we passing this in the queue payload?

Reply:
The payload is an `EventDelivery` object which is a minified version of the actual eventDelivery, so it should still have the `acknowledgedAt` field in case it is needed at the other end.

Reply:
I don't think we need it till we dispatch the event delivery. What if we fetch the ack time from the event delivery's event?

Reply:
Ok. Will remove.